packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -26,19 +27,19 @@ const releaseController = {
   },
 
   async create(ctx: Koa.Context) {
-    const { user } = ctx.state;
-    const { body } = ctx.request;
+    const user: UserInfo = ctx.state.user;
+    const releaseCreateArgs: ReleaseCreateArgs = ctx.request.body;
```

Comment:
i find that wording a bit strange, something like `releaseCreationArgs` or `createReleaseArgs` sounds more readable to me

Reply:
How about just `releaseArgs` we know it's create based on the scope of the function and the type. Do you also find the name of the type confusing 🤔 ?

Reply:
nope that sounds good